### PR TITLE
Automerge version bumps

### PR DESCRIPTION
**Pull request by rail**
This change marks PRs created by `cockroach-teamcity` ready to be merged after all checks are passed.

### CODE REVIEW

File: .github/workflows/automerge.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          # Fetch all branches so it is possible to checkout the default branch files.
+          fetch-depth: 0
+      # The next steps tries to reproduce the steps taken to generate the
+      # files. We restore the changed files to their original state on the
+      # default branch, run the commands to regenerate the change, and verify
+      # the result matches the PR contents (git diff does not show any changes).
+      - name: Regenerate the patch
```
Comment (celiala) [inline .github/workflows/automerge.yaml:33]: nit: maybe name this "confirm expected patch" or something?

(then under "verify nothing changed" move the build steps in there, i.e.)

>  bazel build //build
> $(bazel info bazel-bin)/build/build_/build bump "$appVersion"

**Review by celiala — APPROVED** [review]
woo - excited for this!

had one suggestion (inline) - but not blocking (i.e. PR also okay as-is)

File: .github/workflows/automerge.yaml (hunk 1 above)
Comment (rail) [reply .github/workflows/automerge.yaml:33]: I'll rename the step. 👍 
The build steps generate the patch, so they kind of belong to this step IMO.

Comment (celiala) [reply .github/workflows/automerge.yaml]: ah, gotcha. cool, works for me :) 👍🏼

**Pull request merged**